examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/transformers/weight_only/text-generation/llm_quantization_recipes.md

```diff
@@ -0,0 +1,983 @@
+# Step-by-Step recipes for LLM quantization
+
+This document describes the step-by-step instructions to run large language models (LLMs) on 4th Gen Intel® Xeon® Scalable Processor (codenamed Sapphire Rapids) with [PyTorch](https://pytorch.org/) and [Intel® Extension for PyTorch](https://github.com/intel/intel-extension-for-pytorch).
+
+The scripts [run_generation_sq.py](./run_generation_sq.py) and [run_generation_cpu_woq.py](./run_generation_cpu_woq.py) provide two quantization approaches respectively (SmoothQuant, Weight-Only Quantization) based on [Intel® Neural Compressor](https://github.com/intel/neural-compressor) and return last word prediction accuracy by [lm-evaluation-harness](https://github.com/EleutherAI/lm-evaluation-harness/tree/master).
+
+# Validated Models
+
+|                             Model Name                              |
+| :-----------------------------------------------------------------: |
+|             [EleutherAI/gpt-j-6b](#eleutheraigpt-j-6b)              |
+|                [facebook/opt-1.3b](#facebookopt-13b)                |
+|                [facebook/opt-30b](#facebookopt-30b)                 |
+|        [meta-llama/Llama-2-7b-hf](#meta-llamallama-2-7b-hf)         |
+|       [meta-llama/Llama-2-13b-hf](#meta-llamallama-2-13b-hf)        |
+|       [meta-llama/Llama-2-70b-hf](#meta-llamallama-2-70b-hf)        |
+|               [tiiuae/falcon-40b](#tiiuaefalcon-40b)                |
+|                [tiiuae/falcon-7b](#tiiuaefalcon-7b)                 |
+|  [baichuan-inc/Baichuan2-7B-Chat](#baichuan-inc/Baichuan2-7B-Chat)  |
+| [baichuan-inc/Baichuan2-13B-Chat](#baichuan-inc/Baichuan2-13B-Chat) |
+|  [baichuan-inc/Baichuan-13B-Chat](#baichuan-inc/Baichuan-13B-Chat)  |
+|               [THUDM/chatglm2-6b](#THUDM/chatglm2-6b)               |
+|               [THUDM/chatglm3-6b](#THUDM/chatglm3-6b)               |
+|            [bigscience/bloom-1b7](#bigscience/bloom-1b7)            |
+|         [EleutherAI/gpt-neox-20b](#EleutherAI/gpt-neox-20b)         |
+|       [mistralai/Mistral-7B-v0.1](#mistralai/Mistral-7B-v0.1)       |
+|         [databricks/dolly-v2-12b](#databricks/dolly-v2-12b)         |
+
+# Prerequisite
+
+```bash
+# Installation
+git clone https://github.com/intel/intel-extension-for-transformers.git
+
+# install ITREX
+cd intel-extension-for-transformers
+pip install -r requirements.txt
+pip install -v .
+
+# install requirements
+cd examples/huggingface/pytorch/text-generation/quantization
+pip install -r requirements.txt
+pip install neural-compressor==3.0
+pip install torch==2.3.0+cpu --index-url https://download.pytorch.org/whl/cpu
+# 4.38.1 is only limited by smoothquant
+pip install transformers==4.38.2  # 4.42.4 for mistralai/Mistral-7B-v0.1
+# ipex is only necessary for smoothquant
+pip install intel-extension-for-pytorch==2.3.0
+```
+
+# Run Quantization and evaluate INT8 accuracy
```

Comment:
For INT8 SQ part, it is not in v3.1 scope. if we also port to INC, did we conduct minimal local test?

Reply:
SQ not integrated